# .Net Core 调用异步方法抛异常A second operation started on this context before a previous asynchronous operation completed

### 原因：线程安全问题

### 解决方案：加个锁

